plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -409,11 +409,21 @@ func (c *CiscoTelemetryMDT) handleTelemetry(data []byte) {
 		if content != nil {
 			// Parse values
 			for _, subfield := range content.Fields {
+				if len(subfield.Fields) == 0 {
+					continue
+				}
+
```

Comment:
Hmmm, are you sure that `parseContentField` will always fail if there are not fields or does it in some cases return a sensible value?

Reply:
Ah good point, I may have went a little overboard on the checking. I'll remove this one.